_sass/layout.scss

```diff
@@ -250,177 +250,140 @@ table, th, td {
 	margin: 0px auto;
 	transition-timing-function: linear;
 	transition: cubic-bezier(0, 1, 0, 1);
-  
-  }
-  
-  .innerdiv {
+}
+
+.innerdiv {
 	padding: 10px;
-  }
+}
 
 .toggle input[type="checkbox"]:checked + .hidediv {
 	height: 0;
 	opacity: 0;
 	display: block;
-  }
+}
 
-  .k8s-instructions {
+.k8s-instructions {
 	position: absolute;
-    opacity: 0;
-    pointer-events: none;
-  }
-
+	opacity: 0;
+	pointer-events: none;
+}
 
 nav a:hover {
 	border-bottom-width: 0px !important;
 }
-
-
+#bounce {
+	height: 60px;
+	width: 60px;
+}
 // responsive getting started section
-#bounce:hover{
+#bounce:hover {
 	-ms-transform: scale(1.5); /* IE 9 */
--webkit-transform: scale(1.5); /* Safari 3-8 */
-transform: scale(1.5); 
-
-  }
-  hr{
-	opacity: .4;
+	-webkit-transform: scale(1.5); /* Safari 3-8 */
+	transform: scale(1.5);
+}
+hr {
+	opacity: 0.4;
 	width: 900px;
-  }
-  @media screen and (max-width: 1900px){
-	#box{
-	  width: 40%;
-	  overflow: hidden;
-	  margin-left: -60px;
-	}
-	#containers{
-	  width: 80%;
-	  position: relative;
-	  
-	  left: -100px;
-	}
-
-  }
-
-  @media screen and (max-width: 1440px){
-	#box{
-	  width: 40%;
-	  overflow: hidden;
-	  margin-left: -60px;
+}
+#breakWords {
+	word-break: break-all;
+}
+#containers{
+	position: relative;
+	right: -100px;
+}
+// responsive getting started
+@media screen and (max-width: 1600px){
+	
+	.toggle{
+		margin-left: -215px;
 	}
-	#containers{
-	  width: 80%;
-	  position: relative;
-	  top: -100px;
-	  left: -200px;
+	.login  {
+		padding-left: 215px;
 	}
-
-  }
-  @media screen and (max-width: 1000px){
-	#box{
-	  width: 100%;
+	hr{
+		margin-left: 205px;
+		width: 500px;
 	}
-	#containers{
-	  width: 100%;
-	  height: 50%;
-	  position: relative;
-	  top: 300px;
-	  left: -650px;
-	  width: 60%;
+	#windowsBlock{
+		padding-left: 210px;
 	}
-	.login{
+	
+	#containers {
+		width: 60%;
 		position: relative;
-		top: -340px;
-		width: 40%;
-		left: 450px;
+		top: -10px;
+		
 	}
-	
 
-  }
+}
 
-  @media screen and (max-width: 500px){
-	#box{
-	  width: 100%;
-	  font-size: small;
+@media screen and (max-width: 1200px) {
```

Comment:
@gunjanmimo  Can you add media queries for mobile and tablet view as well? I think that would solve following errors:-

Mobile View:-

![Screenshot from 2020-06-16 11-48-01](https://user-images.githubusercontent.com/35156299/84743120-1bef9a80-afcf-11ea-8687-3812db92da40.png)


Tablet View:-

![Screenshot from 2020-06-16 11-50-22](https://user-images.githubusercontent.com/35156299/84743137-227e1200-afcf-11ea-926b-c1baf00243c0.png)